src/cmds/check_plots.py

```diff
@@ -1,41 +1,68 @@
 import argparse
```

Comment:
Any chance you can port this to under the `chia` umbrella so this command becomes `chia check plots`? Take a look at some of the other chia commands like `init`, `generate` as templates... it's not too hard.

Reply:
Yes I can do that! I just haven't had the time to do it. I don't know if I like the "verb" thing, since all the keys commands for example are related. 
So chia keys add, chia keys remove, etc should all be under the same program no?
Similar to chia plots generate, chia plots check, it seems that nouns make more sense here